aptos-move/aptos-vm-types/src/output.rs

```diff
@@ -108,21 +108,34 @@ impl VMOutput {
         resolver: &impl AggregatorV1Resolver,
     ) -> anyhow::Result<TransactionOutput, VMStatus> {
         let materialized_output = self.try_materialize(resolver)?;
-        debug_assert!(
+        Self::convert_to_transaction_output(materialized_output)
+    }
+
+    /// Same as `try_materialize` but also constructs `TransactionOutput`.
+    pub fn into_transaction_output(self) -> anyhow::Result<TransactionOutput, VMStatus> {
+        let (change_set, fee_statement, status) = self.unpack_with_fee_statement();
```

Comment:
What does unpacking fee statement mean?

Reply:
there is unpack() and unpack_with_fee_statement(). it is just like into_inner() in other places - extracting relevant parts.

we are doing that in try_materialize above in line 93, here we do the same without try_materialize